### ISSUE TITLE
Mouse right button is not emitting mouseUp event

### ISSUE THREAD

**Issue by isybai** [electron/electron#45093]
### Preflight Checklist

- [x] I have read the [Contributing Guidelines](https://github.com/electron/electron/blob/main/CONTRIBUTING.md) for this project.
- [x] I agree to follow the [Code of Conduct](https://github.com/electron/electron/blob/main/CODE_OF_CONDUCT.md) that this project adheres to.
- [x] I have searched the [issue tracker](https://www.github.com/electron/electron/issues) for a bug report that matches the one I want to file, without success.

### Electron Version

33.2.1

### What operating system(s) are you using?

macOS

### Operating System Version

macOS Sequoia 15.2

### What arch are you using?

arm64 (including Apple Silicon)

### Expected Behavior

Mouse right button needs to emit mouseUp event when button is released

### Actual Behavior

Mouse right button is not emitting mouseUp event when button is released

can be reproduced by:
```
 mainWindow.webContents.addListener('input-event', (_, event) => {
    if (event.type.includes('mouse') && !['mouseMove', 'mouseLeave', 'mouseEnter'].includes(event.type)) {
      console.log(event)
    }
  })
```
![Image](https://github.com/user-attachments/assets/166576ea-d026-4cf0-92db-6b060d092658)

**Comment by isybai** [comment]
closing issue, since found that issue is resolved in version 34.0.0-beta.14

**Comment by m-mohamedin** [comment]
Can we please reopen this issue. I'm still having it in v34.0.1 (both using the above code and JS code)

Thanks
